src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate.java

```diff
@@ -150,6 +151,15 @@ public void setEnvVars(List<ContainerEnvVar> envVars) {
         this.envVars.addAll(envVars);
     }
 
+    public List<PortMapping> getPorts() {
+        return ports != null ? ports : Collections.emptyList();
+    }
+
+    @DataBoundSetter
+    public void setPorts(List<PortMapping> ports) {
+        this.ports.addAll(ports);
```

Comment:
@carlossg, I upgrade to a version of the plugin with this patched it and I didn't see any errors or nothing anything once jenkins was up again and running jobs.